---
layout: article
title: 
key: page-about
sidebar:
  nav: docs-en
aside:
  toc: true
---

|--|--|
|![Image](https://loop.frontiersin.org/images/profile/706757/203){:.circle.border.shadow}| Research Associate, [TIGEM (Telethon Institute of Genetics and Medicine)](http://www.tigem.it){:target="_blank"} <br> Research Associate, [University of Naples Federico II](http://www.unina.it){:target="_blank"} <br> Department of Chemical, Materials and Production Engineering <br> TIGEM, Pozzuoli (NA) <br> Via Campi Flegrei 34, 80078 Pozzuoli (NA), Italy  <br> [Tel](tel:003908119230691) - [Mail Me](mailto:gambardella@tigem.it){:target="_blank"} - [ORCID](https://orcid.org/0000-0001-7517-1347){:target="_blank"}|

## Research Interests

Computational Biology, Bioinformatics, Computer Science, Systems Biology, Analysis of Big data in biology

## Education

2013, PhD, Computational Biology and Bioinformatics   
University of Naples, Federico II

2009, M.Sc. degree in Computer Science, 110/110  
University of Naples, Federico II

2006, B.Sc. degree in Computer Science, 103/110  
University of Naples, Federico II 

## Professional Experience

2019 (Jen.) – Current: Research Associate at TIGEM.  
2018 (May) - Current: Junior PI at University of Naples, Federico II.  
2015 (Sep.) – 2018 (May): Post-Doc Scientist at TIGEM in Diego di Bernardo’s Lab.  
2014 (Jan.) – 2015 (August): Post-Doc Scientist at King’s College of London (KCL) in Francesca Ciccarelli’s Lab.  
2013 (July) – 2013 (Dec.): Post-Doc Scientist at European Institute of Oncology (IEO) in Francesca Ciccarelli’s Lab.  


## Publications

### Peer-Reviewed Journal Articles

__G. Gambardella__ and D. di Bernardo - A tool for visualization and analysis of single-cell RNA-seq data based on text-mining. - *Front. Genet. - Statistical Genetics and Methodology (2019)*. [Read more..](https://www.frontiersin.org/articles/10.3389/fgene.2019.00734/abstract){:target="_blank"}.  

__G. Gambardella et al.__ - The impact of microRNAs on transcriptional heterogeneity and gene co-expression across single embryonic stem cells – *Nature Comm. (2017)*. [Read More](https://www.nature.com/articles/ncomms14126){:target="_blank"}

__G. Gambardella et al.__ - MEGA-V: detection of variant gene sets in patient cohorts – *Bioinformatics (2016)*. [Read More](https://academic.oup.com/bioinformatics/article/33/8/1248/2730231){:target="_blank"}

__G. Gambardella et al.__ - Patients with genetically heterogeneous synchronous colorectal cancer have rare damaging germline mutations in immune-related genes – *Nature Comm. (2016)*. [Read More](https://www.nature.com/articles/ncomms12072){:target="_blank"}

__G. Gambardella et al.__ - A reverse-engineering approach to dissect post-translational modulators of transcription factor’s activity from transcriptional data – *BMC Bioinformatics (2015)*. [Read More](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0700-3){:target="_blank"}

C. Racioppi, A. K. Kamal, F. Razy-Krajka, __G. Gambardella__ et al. - Fibroblast growth factor signalling controls nervous system patterning and pigment cell formation in Ciona intestinalis. – *Nature Communications (2014)*. [Read More](https://www.nature.com/articles/ncomms5830){:target="_blank"}

__G. Gambardella et al.__ - Differential Network Analysis for the identification of condition-specific pathway activity and regulation – *Bioinformatics (2013)*. [Read More](https://academic.oup.com/bioinformatics/article/29/14/1776/232944){:target="_blank"}

R. De Cegli, S. Iacobacci, G. Flore, __G. Gambardella__ et al. - Reverse engineering a mouse embryonic stem cell-specific transcriptional network reveals a new modulator of neuronal differentiation - *Nucl. Acids Res. (2013)*. [Read More](https://academic.oup.com/nar/article/41/2/711/1066253){:target="_blank"}


### Book Chapters

__G. Gambardella et al.__ - Differential Equation based reverse-engineering algorithms: pros and cons – *Chapter in Springer book (2013)*. [Read More](https://link.springer.com/chapter/10.1007%2F978-3-642-45161-4_4)  

## Funded Grants

2018-2020, Junior PI, Single-cell transcriptomics to characterize breast tumor sub-types and to identify novel biomarkers of drug sensitivity  (100.000€ in total). In the ambit of [STAR (Sostegno Territoriale alle Attività di Ricerca)](http://www.coinor.unina.it/programmastar/){:target="_blank"} project from Compagnia San Paolo and University of Naples Federico II.

## Fellowships and Awards

2018-2019, Post-Doctoral Fellowship from [Fondazione Veronesi](https://www.fondazioneveronesi.it/ricerca/i-nostri-ricercatori/gennaro-gambardella){:target="_blank"}.

2016 (Jen.), Best poster at Festival of genomics in London. “A genomic approach to study synchronous colorectal cancer”.  

2010 (Jul.) Fellowship for the PASCAL (Pattern Analysis, Statistical modelling and ComputAtional Learning) bootcamp in Marseille.

## Presentations 

### Conference Presentations

2013 (Nov. 8-12)	Selected to talk at [RECOMB/DREAM 2013](http://www.iscb.org/recomb-regsysgen2013){:target="_blank"} Systems Biology conference (Canada, Toronto). Title of talk: *A computational high-throughput approach to discover post-translational modulators of transcription factor activity from gene expression profiles*.  

2012 (November):	Presentation at [RECOMB/DREAM 2012](http://recomb-2012.c2b2.columbia.edu/){:target="_blank"} (US, San Francisco). *Identification of Post-Translational modifications from gene expression profiles using Multi-Information*.  

2010 (November):	Selected to talk at [RECOMB/DREAM 2010](http://recombsat.c2b2.columbia.edu/){:target="_blank"} for the Systems Biology conference (US, NY, Columbia University). Title of talk: *Identification and Analysis of tissue-specific co-expression networks from human samples*.  

### Lectures & Workshops

2019 (June 14)	Presentation at IIGM (Italian Institute for Genomic Medicine) for [Single Cell Revolution 2.0 workshop](https://www.iigm.it/oggetti/1573_SingleCellMeeting.pdf){:target="_blank"}. Title of the talk: *Single-cell transcriptomics to reveal biomarkers and mechanisms of drug sensitivity in breast cancer cells*.  

2019 (May 9)	Lecture at TIGEM for an [Elixir Training Course](https://elixir-iib-training.github.io/website/2019/05/07/SingleCell-Pozzuoli.html){:target="_blank"}: *Single-Cell RNA Sequencing and Data Analysis*. Presentation can be found [HERE](https://elixir-iib-training.github.io/2019-05-07-pozzuoli-singlecell/pres/Gambardella_scRNA_Clustering_and_Visualization.pdf){:target="_blank"}.  

2018 (Nov. 22)	Lecture at Istituto Superiore di Sanità: *In-silico Models for Oncology*.  

2014 (May 13)	Lecture at The Systems Biology Modelling Cycle (EBI, UK, Cambridge, supported by BioPreDyn): *Inference methods for biological networks*.  

2013 (April 22):	Lecture at EPIGEN course (TIGEM, Italy, Naples): *Introduction to reverse-engineering methods*. 

2011 (July):	Lecture at Lifelong Learning Programme Erasmus (UNISA, Italy, Salerno) - *Interdisciplinary Approaches to Microarray Data Analysis*.  

## Software Developed

### R packages

G. Gambardella and D. di Bernardo - A tool for visualization and analysis of single-cell RNA-seq data based on text-mining. - *Front. Genet. - Statistical Genetics and Methodology (2019)*. [R package](https://github.com/dibbelab/gficf).  

G. Gambardella et al. - MEGA-V: detection of variant gene sets in patient cohorts – *Bioinformatics (2016)*. [R package](https://github.com/ciccalab/MEGA){:target="_blank"}.


### Other Tools

G. Gambardella et al. - Differential Network Analysis for the identification of condition-specific pathway activity and regulation – *Bioinformatics (2013)*. [Use the tool](http://dina.tigem.it){:target="_blank"}.

G. Gambardella et al. - A reverse-engineering approach to dissect post-translational modulators of transcription factor’s activity from transcriptional data – *BMC Bioinformatics (2015)*. [Use the tool](http://dmi.tigem.it){:target="_blank"}.
