---
title: "GF-ICF pubblished"
tags: gf-icf
sidebar:
  nav: docs-en
---

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-144257957-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-144257957-1');
</script>

## Abstract
Gene expression in individual cells can now be measured for thousands of cells in a single experiment thanks to innovative sample-preparation and sequencing technologies. State-of-the-art computational pipelines for single-cell RNA-sequencing data, however, still employ computational methods that were developed for traditional bulk RNA-sequencing data, thus not accounting for the peculiarities of single-cell data, such as sparseness and zero inflated counts. Here, we present a ready-to-use pipeline named gf-icf (Gene Frequency â€“ Inverse Cell Frequency) for normalisation of raw counts, feature selection and dimensionality reduction of scRNA-seq data for their visualization and subsequent analysis. Our work is based on a data transformation model named Term Frequency - Inverse Document Frequency (TF-IDF), which has been extensively used in the field of text-mining where extremely sparse and zero-inflated data are common. Using benchmark scRNA-seq datasets, we show that the gf-icf pipeline outperforms existing state-of-the-art methods in terms of improved visualisation and ability to separate and distinguish different cell types.

[Read more..](https://www.frontiersin.org/articles/10.3389/fgene.2019.00734/abstract){:target="_blank"} about GF-ICF on Front. Genet. - Statistical Genetics and Methodology journal 
